test/server/cards/03_TWI/units/RepublicAttackPod.spec.ts

```diff
@@ -0,0 +1,42 @@
+describe('Republic Attack Pod', function() {
+    integration(function(contextRef) {
+        describe('Republic Attack Pod\'s decrease cost ability', function() {
+            it('should cost 1 less if there is 3 friendly units', () => {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['republic-attack-pod'],
+                        groundArena: ['fleet-lieutenant', 'battlefield-marine', 'pyke-sentinel'],
+                        leader: 'fennec-shand#honoring-the-deal',
+                        resources: 6
+                    },
+                    player2: {
+                        groundArena: ['greedo#slow-on-the-draw', 'rey#keeping-the-past'],
+                    }
+                });
+
+                const { context } = contextRef;
+
+                const reset = () => {
+                    context.player2.passAction();
+                    context.player1.moveCard(context.republicAttackPod, 'hand');
```

Comment:
Similar note here - better to use Waylay to return the card to hand in between tests. This is also a situation where I'm potentially fine having multiple cases, since it's harder to reset state in between than it is with attacks. It's up to personal preference though.